src/qml/pages/wallet/CreateIntro.qml

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import "../../controls"
+import "../../components"
+import "../settings"
+
+Page {
+    id: root
+    background: null
+
+    header: NavigationBar2 {
+        id: navbar
+        leftItem: NavButton {
+            iconSource: "image://images/caret-left"
+            text: qsTr("Back")
+            onClicked: {
+                root.StackView.view.pop()
+            }
+        }
+    }
+
+    ColumnLayout {
+        id: columnLayout
+        width: Math.min(parent.width, 450)
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        Item {
+            Layout.alignment: Qt.AlignCenter
+            Layout.preferredHeight: circle.height
+            Layout.preferredWidth: circle.width
+            Rectangle {
+                id: circle
+                width: 60
+                height: width
+                radius: width / 2
+                color: Theme.color.blue
+            }
+            Icon {
+                source: "image://images/wallet"
+                color: Theme.color.white
+                width: 30
+                height: width
+                anchors.centerIn: circle
+            }
+        }
+
+        CoreText {
+            Layout.topMargin: 25
+            Layout.alignment: Qt.AlignCenter
+            text: qsTr("You are about to create \na single-key bitcoin wallet")
```

Comment:
```suggestion
            text: qsTr("You are about to create \na single-sig bitcoin wallet")
```

Reply:
It says "single-key" also in [figma](https://www.figma.com/design/ek8w3n3upbluw5UL2lGhRx/Bitcoin-Core-App-Design?node-id=7516-13170).

Reply:
It says "single-key" also in [figma](https://www.figma.com/design/ek8w3n3upbluw5UL2lGhRx/Bitcoin-Core-App-Design?node-id=7516-13170).

Reply:
yeah, "single-key" is wrong and also confusing imo

Reply:
I concur "single-sig" is more intuitive than "single-key". It seems "single-key" is referenced in the docs although most people/users only refer to "single-sig"

Reply:
We generally use multi-key in the design community ([example](https://bitcoin.design/guide/how-it-works/private-key-management/multi-key/)). Some reasons:

- "sig" is an abbreviation that many people may not be familiar with
- Avoiding "signature" altogether in the applications removes another technical concept that people don't really need to be aware of. For example, "sign transaction" can be just be the more common term "approve transaction". Just part of the overall idea to use fewer technical terms if possible
- If we want to be super specific in technical terms, then a "single-sig" wallet could even be a "multi-key" wallet. For example, in a wallet with 3 keys that only requires 1 signature

I understand that the technical terms can be more intuitive for us who are super deep into the tech, and they allow us to discuss things more precisely. For broader audiences, especially global ones where english may be the second language, it's often best to simplify things a little (you even have tools for that like [Hemingway](https://hemingwayapp.com)). Does that sound reasonable?